container/BUILD

```diff
@@ -64,14 +64,21 @@ py_binary(
     ],
 )
 
+py_library(
```

Comment:
could you please try the alternative of having a http_file to pull the archive.py file from Bazel repo?